src/graphql/resolvers/mockresolvers/mockDb/mockUtils.ts

```diff
@@ -23,3 +24,17 @@ type Entries<T> = {
 export function entries<T extends Record<string, unknown>>(obj: T): Entries<T> {
     return Object.entries(obj) as unknown as Entries<T>
 }
+
```

Comment:
kanskje bare fjerne filtrerte spørsmål fra testdataen isteden for å duplisere dette i mockingen? :smile: